<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtQQ_ServerClass</class>
 <widget class="QDialog" name="QtQQ_ServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtQQ_Server</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>698</width>
      <height>498</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>700</width>
       <height>500</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>702</x>
     <y>1</y>
     <width>296</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>r4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>293</width>
      <height>295</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>299</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户部门:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户姓名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户头像:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="employeeDepBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>人事群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>研发群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>市场群</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectPictureBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>40</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新增用户</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>用户头像</string>
     </property>
    </widget>
    <widget class="QComboBox" name="departmentBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>公司群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人事群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>研发群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>市场群</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="queryDepartmentBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查询用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queryIDBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>215</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查询用户</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="queryIDLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>215</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>注销用户</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="logoutIDLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户ID</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtQQ_Server.qrc"/>
 </resources>
 <connections/>
</ui>
